doc/manual/documentation.rst

```diff
@@ -30,7 +30,7 @@ macros and pass them to the ``@doc`` macro just as well.
 
 Here is a more complex example, still using Markdown:
 
-.. doctest::
+.. code-block:: julia
```

Comment:
I changed this to a code block as Julia interprets `function bar(x, y) ...` as a function call to `bar` which raises an exception.